jdaviz/configs/specviz2d/plugins/spectral_extraction/spectral_extraction.py

```diff
@@ -260,6 +260,9 @@ def __init__(self, *args, **kwargs):
                                       default_text='From Plugin',
                                       filters=['is_trace'])
 
+        # Cache the actual calculated trace
```

Comment:
Can you please stress in an in-line comment (here and for the other cache) that this should only ever be used for the interactive previews and should always be recomputed when the user is calling a method (since the cache could be out of date if the user is updating traitlets with the plugin inactive).

Reply:
Thanks!